packages/Takeover/stories/TakeoverStory.js

```diff
@@ -17,12 +17,12 @@ const FooterButtonGroup = styled.div`
 
 export const CustomFooterLayout = () => (
   <FooterButtonGroup>
-    <Button kind={Button.types.kind.MINOR} size="medium">
+    <Button kind={Button.types.kind.MINOR} size="large">
```

Comment:
Ah sorry I was wondering if we can use `Button.types.size.LARGE` instead of hardcoded `large`? Same thing goes to Footer.js